specification/introduction/index.md

```diff
@@ -42,16 +42,14 @@ As an interoperability standard, it must be possible to use the CS IP regardless
 
 To allow for such in-depth control over specific content types and formats, E-ARK specifications introduce the concept of Content Information Type Specifications.  A Content Information Type Specification can include detailed requirements on how content, metadata, and documentation for specific content types (for example relational databases or geospatial data) have to be handled within a CS IP (or E-ARK SIP, AIP or DIP).
 
-For now (February 2017) there are seven Content Information Type Specifications which have been created by the E-ARK project and have been verified for usage within the Common Specification for Information
+For now (November 2018) these Content Information Type Specifications which have been created by the E-ARK project and enchanced by the DILCIS Board have been verified for usage within the Common Specification for Information
```

Comment:
As of November 2018 these Content Information Type Specifications, created by the E-ARK project and enchanced by the DILCIS Board, have been verified for usage within the Common Specification for Information